### PR TITLE
Easier database initialization

### PR DESCRIPTION
**Pull request by yakloinsteak**
## Description

Initializes the databases on first deployment

## Type of change
- [x] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

**Comment by yakloinsteak** [comment]
@eanders @joe4123 After this builds and Elliot takes a look, we'll deploy to triad staging. I took a stab at what it means for seeds to be initialized and how to detect a new database. Those are the important parts to review.

File: bin/db_prep [hunk 1]
```diff
@@ -5,56 +5,139 @@
...
+  section = db(db_section)
+  conn = PG.connect(**section)
+  if db_section == "primary"
+    result = conn.exec_params('SELECT count(*) FROM translations').entries
+    num = result[0]["count"].to_i
+    num > 10
+  elsif db_section == "warehouse"
+    result = conn.exec_params('SELECT count(*) FROM grades').entries
+    num = result[0]["count"].to_i
+    num > 1
```
Comment (eanders) [inline bin/db_prep]:
```suggestion
    result = conn.exec_params('SELECT count(*) FROM data_sources').entries
    num = result[0]["count"].to_i
    num > 0
```

File: bin/db_prep [hunk 2]
```diff
@@ -5,56 +5,139 @@
...
 end
 
-dont_create_databases = (ARGV[0] == '--no-create-databases')
+def seeds_ran?(db_section)
+  section = db(db_section)
+  conn = PG.connect(**section)
+  if db_section == "primary"
+    result = conn.exec_params('SELECT count(*) FROM translations').entries
+    num = result[0]["count"].to_i
+    num > 10
```
Comment (eanders) [inline bin/db_prep]:
```suggestion
    result = conn.exec_params('SELECT count(*) FROM collections').entries
    num = result[0]["count"].to_i
    num > 0
```

**Review by eanders — APPROVED** [review]
Thanks @yakloinsteak !  Looks great, just a couple suggestions inline for what to check for seeds.

**Comment by yakloinsteak** [comment]
@eanders I've deployed this three times, and I think it's ready for running on staging elsewhere and/or the new production environment.

**Pull request merged**